lib/python/examples/fwdllm/expts/run_tc_expts/json_scripts/aggregator.json

```diff
@@ -101,11 +102,18 @@
         "uri": ""
     },
     "selector": {
-        "sort": "random",
-        "kwargs": {
-            "k": 10,
-            "minInitialTrainers": 50
-        }
+    "sort": "async_oort",
+    "kwargs": {
+        "//": "c: concurrency level",
+        "c": 30,
+        "aggGoal": 10,
+        "evalGoalFactor": 0.5,
+        "selectType": "default",
+        "roundNudgeType": "last_eval",
+         "minInitialTrainers": 5,
+         "k": 10,
+         "is_async": true
```

Comment:
`async_oort` selector should be async only right, so why is another field called `is_async` needed?